docs/quickstart.rst

```diff
@@ -71,26 +65,8 @@ Now, let's scaffold the instance using the `official cookiecutter template
 .. code-block:: shell
 
   (my-repository-venv)$ cookiecutter gh:inveniosoftware/cookiecutter-invenio-instance --checkout v3.0
-  project_name [My site]: My repository
-  project_shortname [my-repository]:
-  project_site [my-repository.com]:
-  package_name [my_repository]:
-  github_repo [my-repository/my-repository]:
-  description [Invenio digital library framework.]: My repository digital library
-  author_name [CERN]: Me
-  author_email [info@inveniosoftware.org]: me@my-repository.com
-  year [2018]:
-  copyright_holder [Me]:
-  transifex_project [my-repository]:
-  Select database:
-  1 - postgresql
-  2 - mysql
-  Choose from 1, 2 [1]:
-  Select elasticsearch:
-  1 - 6
-  2 - 5
-  Choose from 1, 2 [1]:
-  # ... project scaffolding output ...
+  project_name [My site]: ...
+  # continuing with the defaults ...
```

Comment:
The renaming from my-site to my-repository takes up ~20 lines, we could mention that we are continuing with the defaults and rename my-repository to my-site elsewhere in the docs?